app/browser/dataset-browse.tsx

```diff
@@ -1015,7 +1015,7 @@ export const DatasetResults = ({
   }
 
   return (
-    <>
+    <div>
```

Comment:
Must this be a div?

Reply:
Yes, otherwise `&:not(:first-child)` CSS selector can't target a proper element 🥹